App/config/services.yaml

```
@@ -9,11 +9,17 @@ services:
   App\Command\DatabaseFlushTest: ~
   
   App\Controller\GroupController: ~
+  App\Controller\RegisterController: ~
```

Following [the beginning of the documentation](https://symfony.com/doc/current/controller/service.html), you can add a tag to tell Symfony those are controllers

```suggestion
  App\Controller\:
    resource: '%kernel.project_dir%/src/Controller/**/*Controller.php'
    tags: ['controller.service_arguments']
```

---

Done.